packages/design-system/src/components/VerticalNav/VerticalNavItemLabel.tsx

```diff
@@ -55,8 +55,8 @@ export const VerticalNavItemLabel = (props: VerticalNavItemLabelProps): React.Re
 
   let otherProps;
   if (LabelComponent === 'button') {
-    const collapsedLabel = props.ariaCollapsedStateButtonLabel ?? t('verticalNav.expand');
-    const expandedLabel = props.ariaExpandedStateButtonLabel ?? t('verticalNav.collapse');
+    const collapsedLabel = props.ariaCollapsedStateButtonLabel;
+    const expandedLabel = props.ariaExpandedStateButtonLabel;
```

Comment:
These could be inlined in the one place they're used now

Reply:
i just realized this is unnecessary to include - this sets the `title` attribute on a button and titles aren't accessible so this does nothing and should be removed